### PR TITLE
#101 adding rich text editor to questions description

### PR DESCRIPTION
**Pull request by ghost**
#101 adding rich text editor to questions description

### CODE REVIEW

File: App/PHP_HTML/question_submission.php [hunk 1]
```diff
@@ -15,18 +15,19 @@
...
+	
     $sql = "INSERT INTO questions (question_by, question_title, question_description, question_date, question_by_user) VALUES ($user_id, \"$title\", \"$details\", NOW(), '$question_by_user')";
     $db->query($sql);
 
     // increase users question count
     $sql = "UPDATE users SET user_questions_count = user_questions_count + 1 WHERE user_id = '".$user_id."'";
     $db->query($sql);
 
-    $sql = "SELECT question_id FROM questions WHERE question_by='$user_id' AND question_title = '$title'";
+    $sql = "SELECT max(question_id) FROM questions WHERE question_by='$user_id' AND question_title = '$title'";
```
Comment (refatK) [inline App/PHP_HTML/question_submission.php]: Could you explain this?

**Review by refatK — APPROVED** [review]

File: App/PHP_HTML/question_submission.php (hunk 1 above)
Comment (sbharti123) [reply App/PHP_HTML/question_submission.php]: If we create a question with title matching an existing question, then the old question will be displayed. The max function will show the latest one we created.

File: App/PHP_HTML/answer.php [hunk 2]
```diff
@@ -171,14 +174,16 @@ function AnswerIncrementDislike(answerId) {
 
             echo "<input id='modify-question' class='question-form-button' type='button' value='Modify' onclick='fillForm()'><br><br>";
             $question_action = "question_modify.php?questionId=$qus_id";
-            include('ask_question.php');
+            include('ask_question1.php');
```
Comment (MattYu) [inline App/PHP_HTML/answer.php]: Hi, great work!
Please revise this: ask_question1.php does not exists. The original php file on main has the correct name

Comment (refatK) [reply App/PHP_HTML/answer.php]: This is ok. I fix this on my branch. I also renamed it.

Comment (MattYu) [reply App/PHP_HTML/answer.php]: I'll merge it now, but we should rename it to follow conventions. We lost points on spring 3 due to a similar file name

**Pull request merged**